mock-thunderhead/src/main/resources/application.yml

```diff
@@ -29,6 +29,7 @@ auth:
     mediumdutysdx.enable: true
     upgrade.internalrepo.enable: true
     hbase.cloudstorage.enable: true
+    datalake.efs.enable: true
```

Comment:
this means that locally we will always use EFS. This is what you want to achieve here ?

Reply:
I can turn it off, and only enable in my local dev.